src/request-permission-polyfil.ts

```diff
@@ -1,6 +1,6 @@
-(DeviceMotionEvent as any).requestPermission = () => {
-    return new Promise((resolve) => {
-        (window as any).grantMotionPermission = () => resolve('granted');
-        (window as any).denyMotionPermission = () => resolve('denied');
-    });
-}
\ No newline at end of file
+// (DeviceMotionEvent as any).requestPermission = () => {
```

Comment:
either remove the file completely or keep it uncommented. There is no point of having a commented file in the repo.

My suggestion is to uncomment this and keep it.